site/content/docs/main/restore-reference.md

```diff
@@ -169,6 +169,39 @@ data:
   # class name.
   <old-storage-class>: <new-storage-class>
 ```
+### Changing Pod/Deployment/Statefulsets/Daemonset/Replicaset/Replicationcontroller/Job/Cronjob Image Repositories  
```

Comment:
Since you're listing the "kind" value in the heading, this should be:
```
### Changing Pod/Deployment/StatefulSet/DaemonSet/ReplicaSet/ReplicationController/Job/Cronjob Image Repositories  
```

Reply:
@sseago  thanks so much for your careful review.
I will update it according to all the comments.